docs: add precisions in readme (#275)
Orange-OpenSource/ods-charts: README.md

- [Versioning](#versioning)
- [Copyright and license](#copyright-and-license)

> ## Quick start for users

Reviewer: Here, we could keep "Quick start"

> The ODS Chart library is to be used with Apache ECharts to produce charts in line with the Orange Design System.

Reviewer: ```suggestion
The ODS Charts library is to be used with Apache ECharts to create charts that adhere to the Orange Design System guidelines..
```

> It has been designed to be minimally intrusive and not to interfere with the use of the Apchae Echarts library. The Apache Echarts documentation remains the reference for configuring graphics.

Reviewer: ```suggestion
It has been designed to be minimally intrusive and not to interfere with the use of the Apache ECharts library. The Apache ECharts documentation remains the reference for configuring charts.
```

For example, an Apache Echarts user will initialize a graph with code like this:

> ```

Reviewer: Precise the language for syntax highlighting

var div = document.getElementById('barLine_chart');
var myChart = echarts.init(div, undefined, {

>   renderer: 'svg', // we prefer svg renderer

Reviewer: ```suggestion
  renderer: 'svg', // SVG is recommended
```

});
// Display the chart.
myChart.setOption(dataOptions);
```

> Adding the theme managed by ODS Charts will then consist of doing :

Reviewer: ```suggestion
Adding the theme managed by ODS Charts will then consist of doing:
```

> ```

Reviewer: You can precise the language here for syntax highlighting

> // Build the theme

Reviewer: ```suggestion
// build the theme
```

to be consistent with the other comments

var themeManager = ODSCharts.getThemeManager({
  // you can configure here some optional options
});

// register this theme to echarts
echarts.registerTheme(themeManager.name , themeManager.theme);

var div = document.getElementById('barLine_chart');
var myChart = echarts.init(div, 

>   themeManager.name,  // initialized the chart with the generated theme 

Reviewer: ```suggestion
  themeManager.name,  // initialize the chart with the generated theme 
```

  {

>     renderer: 'svg', // we still prefer svg renderer

Reviewer: ```suggestion
    renderer: 'svg', // SVG rendered is recommended
```

  }
);

// Enriched dataOptions to complete implementation of the Orange Design System.
myChart.setOption(themeManager.getChartOptions(dataOptions));  

```

> The themeManager returned by getThemeManager() can be used to add other features supported by the ODS Charts library. Please refer to [the online documentation for more details](https://ods-charts.netlify.app/). 

Reviewer: ```suggestion
The `themeManager` returned by `getThemeManager()` can be used to add other features supported by the ODS Charts library. Please refer to [the online documentation for more details](https://ods-charts.netlify.app/). 
```

> ## Quick start for contributors

Reviewer: I think this part doesn't really work with what's inside

> - Install with [npm](https://www.npmjs.com/): `npm install ods-charts`
> Read the [documentation](https://ods-charts.netlify.app/) for information on the library content and examples.

It's for the users too.

We could keep just a "Quick start" and mix both contents

Author: Agree. I think it is the clone repo that was useless. I 'll made a proposition

Reviewer: Yes, we can drop the `git clone` part as it's only useful for contributors, and is already mentioned here: https://github.com/Orange-OpenSource/ods-charts/blob/main/CONTRIBUTING.md#pull-requests

> Several quick start options are available:

Reviewer: So there's only one option now, isn't it?

Reviewer: We reformulated it with @louismaximepiton

- npm installation:
  - Install with [npm](https://www.npmjs.com/): `npm install ods-charts`

>   - Use in your js files : `import * as ODSCharts from 'ods-charts';`

Reviewer: ```suggestion
  - Use in your JavaScript files: `import * as ODSCharts from 'ods-charts';`
```

Reviewer 2: Maybe here we could redirect to https://apache.github.io/echarts-handbook/en/basics/import/

> - Add javascript tag in your HTML page:

Reviewer: ```suggestion
- Add JavaScript script in your HTML page:
```

```html

> <script src=" https://cdn.jsdelivr.net/npm/echarts@latest/dist/echarts.min.js "></script>

Reviewer: Are we sure we want to recommend this approach in terms of eco-design? Users should probably only load what they need, and not the entire Apache ECharts library.

Reviewer 2: Maybe not recommendation but it's still a way to achieve it

Author: I can remove reference to cdn.jsdeliver.net and replace it by a [path to the echarts.min.js]

and keep the npm install for both methods

Author: [jsDelivr not recomended](https://github.com/Orange-OpenSource/ods-charts/pull/275/commits/4271b0aa85ed5f75f9aaff35d3c75d4e40ea10b4)

Reviewer 2: Yes I agree with you (even more since we are still using it in the doc :eyes:)

Reviewer: We changed a little bit the wording with @louismaximepiton to explain it
